web/cypress/integration/plugin/schema-smocktest.spec.js

```diff
@@ -47,35 +50,53 @@ context('smoke test for plugin schema', () => {
             .within(() => {
               cy.contains('Enable').click({
                 force: true,
+                timeout
               });
             });
 
           // NOTE: wait for the Drawer to appear on the DOM
-          cy.wait(800);
-          const switchSelector = '#disable';
-          cy.get(switchSelector).click();
+          const drawerSelector = '.ant-drawer-content'
```

Comment:
why do we need to store the value in a variable first?

how about this style? `cy.get('.ant-drawer-content').within`

Reply:
no, we are going to use a public file to store all reuseable selectors in the future.